### PR TITLE
docs(dev): add an initial CHEATSHEET with clickhouse testing detail

### PR DESCRIPTION
**Pull request by hazzadous**
Perhaps we can add some more here over time, although it would be nice
not to make this a crutch where would could easily just make things work

### CODE REVIEW

**Comment by macobo** [comment]
WDYT about have this in the handbook instead? We already have some things under https://posthog.com/handbook/engineering/development-process#how-to-test and other handbook pages

File: DEVELOPER_CHEATSHEET.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Quick reference to common development tasks
+
+## ClickHouse debugging
+
+### How do I see which queries are sent to ClickHouse
+
+In tests, run:
+
+```
+pytest <path-to-test> --log-cli-level=DEBUG
```
Comment (macobo) [inline DEVELOPER_CHEATSHEET.md:10]: You can pass arguments to bin/tests as well

Comment (hazzadous) [reply DEVELOPER_CHEATSHEET.md:10]: Yup, I typically don't use tbh but have added to https://github.com/PostHog/posthog.com/pull/2148 instead. I'll close this PR

**Comment by hazzadous** [comment]
> WDYT about have this in the handbook instead? We already have some things under https://posthog.com/handbook/engineering/development-process#how-to-test and other handbook pages

Yeah I'll just do this. Have added something to https://github.com/PostHog/posthog.com/pull/2148

**Pull request closed without merging**